echo-model/src/main/java/com/netflix/spinnaker/echo/model/pubsub/MessageDescription.java

```diff
@@ -37,6 +38,13 @@
 
   private String messagePayload;
 
+  /**
+   * Optional, additional message metadata sent from the pub/sub system.
+   *
+   * May be null.
+   */
+  private Map<String, Object> messageAttributes;
```

Comment:
Given that payload is a string, should we leave this as a string as well? Seems it will be more kafka-friendly when that time comes (that uses an 8 bit flag object)